velox/functions/prestosql/tests/HyperLogLogFunctionsTest.cpp

```diff
@@ -50,6 +50,38 @@ class HyperLogLogFunctionsTest : public functions::test::FunctionBaseTest {
     return serialized;
   }
 
+  // Helpers to create HLL for testing
```

Comment:
This comment is not very informative. Please, explain what this method actually does.